Classes/Bookmark/BookmarkModel.swift

```diff
@@ -18,14 +18,14 @@ final class BookmarkModel: NSObject, NSCoding, Filterable {
         case title
         case hasIssueEnabled
     }
-    
+
```

Comment:
Not sure if whatever rule is causing this, is going to get super annoying during development?

Reply:
Could disable the rule? It's basically mitigated by an Xcode setting but could be super super annoying for new contributors
![screen_shot_2017-10-23_at_11_47_27_am](https://user-images.githubusercontent.com/3388381/31898802-fb5a33de-b7e7-11e7-8b2c-997bb6765e72.png)

Reply:
Yeah especially since that's not per-project, I'd rather disable it tbh!

Reply:
@Sherlouk should be disabled now https://github.com/rnystrom/GitHawk/pull/704/files#diff-e9e9dfc22d31913c2e705d4d571d54c7R10